### PR TITLE
electrocute_act now stuns for 4 seconds, not 10, and adds convulsing

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
title

## Why It's Good For The Game
@kriskog asked me for this, part of removingducing hardstuns from the game and implementing the effect in more things, to make up for it's removal. 

## Changelog
:cl:
tweak: being electrocuted by things now stuns for 4 seconds, not 10, and adds convulsing
/:cl:

### CODE REVIEW

**Comment by Arianya** [comment]
what does convulsing exactly do

consider increasing the electrocution damage cap or the like - this makes hacking doors a lot safer.

**Comment by carlarctg** [comment]
> what does convulsing exactly do
> 
> consider increasing the electrocution damage cap or the like - this makes hacking doors a lot safer.

it makes you drop inhand items randomly for 15 seconds

**Comment by HomocidlePancakes** [comment]
Electrocution cap is at 190 afair.

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -240,18 +240,17 @@
...
 	if(should_stun)
 		Paralyze(40)
+	apply_status_effect(STATUS_EFFECT_CONVULSING)
 	//Jitter and other fluff.
-	jitteriness += 1000
+	jitteriness += 1000 //apparently jitteriness duration and how much your character shakes around are intertwined, so that's some good coding there (nov2019)
 	do_jitter_animation(jitteriness)
 	stuttering += 2
-	addtimer(CALLBACK(src, .proc/secondary_shock, should_stun), 20)
+	addtimer(CALLBACK(src, .proc/remove_jitter), 20)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/carbon_defense.dm]: there's var callbacks use those.

**Pull request closed without merging**